### PR TITLE
Documentation updates for Dockerfile and README files

### PR DESCRIPTION
**Pull request by mrrajan**
Signed-off-by: Rajan Ravi <rravi@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,31 +33,31 @@ When contributing to this repository, please first discuss the change you wish t
...
   ```
 
-- Test your PR in Jenkins using the rhods-ci-pr-test pipeline
-   - https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test
-   - Log in if required
-   - Build with Parameters (if you don't see this option contact the QE team)
-     - Set the PR id (e.g. 42) in ODS_GIT_REPO_PULL_REQUEST_ID
-     - Select TEST_CLUSTER
-     - Build
+#- Test your PR in Jenkins using the rhoda-ci-pr-test pipeline
```
Comment (mrrajan) [inline CONTRIBUTING.md]: commented this block for now, which will be included when we defined our pipeline

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,31 +33,31 @@ When contributing to this repository, please first discuss the change you wish t
...
 
-- Send a PR to ods-ci using GitHub's web interface
+- Send a PR to rhoda-ci using GitHub's web interface
 
 - If the PR can't be merged, rebase your branch to master:
   ```
-  git remote add upstream https://github.com/red-hat-data-services/ods-ci
+  git remote add upstream https://github.com/RHODA-lab/rhoda-ci.git
   git fetch upstream
   git checkout add-test-ods-542-alerts
```
Comment (ikanias) [inline CONTRIBUTING.md]: I think this should be fixed too right?

File: build/README.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 
 A [Dockerfile](Dockerfile) is available for running tests in a container.
-The latest build can be downloaded from: https://quay.io/odsci/ods-ci:latest
+The latest build can be downloaded from: https://quay.io/rhodaci/rhoda-ci:latest
 
-eg: podman pull quay.io/odsci/ods-ci:latest
+eg: podman pull quay.io/rhodaci/rhoda-ci:latest
 
 ```bash
 # Build the container (optional if you dont want to use the latest from quay.io/odsci)
```
Comment (ikanias) [inline build/README.md]: Should write:
# Build the container (optional if you dont want to use the latest from quay.io/rhoda-ci)

File: build/README.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 ```
 
-### Running the ods-ci container image in OpenShift
+### Running the rhoda-ci container image in OpenShift
 
-After building the container, you can deploy the container in a pod running on OpenShift. You can use [this](./ods-ci.pod.yaml) PersistentVolumeClaim and Pod definition to deploy the ods-ci container.  NOTE: This example pod attaches a PVC to preserve the test artifacts directory between runs and mounts the test-variables.yml file from a Secret.
+After building the container, you can deploy the container in a pod running on OpenShift. You can use [this](./ods-ci.pod.yaml) PersistentVolumeClaim and Pod definition to deploy the rhoda-ci container.  NOTE: This example pod attaches a PVC to preserve the test artifacts directory between runs and mounts the test-variables.yml file from a Secret.
 
 ```
 # Creates a Secret with test variables that can be mounted in ODS-CI container
```
Comment (ikanias) [inline build/README.md]: The above line should say:
# Creates a Secret with test variables that can be mounted in RHODA-CI container

File: build/rhoda-ci.pod.yaml [hunk 5]
```diff
@@ -24,25 +24,25 @@ spec:
 
     - image: <INSERT URL TO YOUR BUILT IMAGE>
       imagePullPolicy: Always
-      name: ods-ci
+      name: rhoda-ci
       env:
         # Use this environment variable to pass args to the ods-ci run script in the container
```
Comment (ikanias) [inline build/rhoda-ci.pod.yaml]: The comment should read:
# Use this environment variable to pass args to the rhoda-ci run script in the container

**Review by ikanias — COMMENTED** [review]
Overall looks good. Just need to update the commented titles (check my comments inside).

**Review by ikanias — COMMENTED** [review]
A small change in the commented title on line 22.

**Comment by ikanias** [comment]
LGTM

**Pull request merged**